bitnami/nginx/README.md

```diff
@@ -294,11 +294,85 @@ In addition, you can also set an external ConfigMap with the configuration file.
 
 In some scenarios, you may require users to authenticate in order to gain access to protected resources. By enabling LDAP, NGINX will make use of an Authorization Daemon to proxy those identification requests against a given LDAP Server.
 
+```
+                ┌────────────────┐           ┌────────────────┐           ┌────────────────┐
+                │     NGINX      │  ----->   │     NGINX      │  ----->   │      LDAP      │
+                │     server     │  <-----   |  ldap daemon   │  <-----   |     server     │
+                └────────────────┘           └────────────────┘           └────────────────┘
+
+```
+
 In order to enable LDAP authentication you can set the `ldapDaemon.enabled` property and follow these steps:
 
-1. Use the `ldapDaemon.nginxServerBlock` property to provide with an additional server block that will make NGINX such a proxy (see `values.yaml`). Alternatively, you can provide this configuration using an external Secret and the property `ldapDaemon.existingNginxServerBlockSecret`.
+1. You have to make your NGINX server aware of the proxy. In order to do so, use the `ldapDaemon.nginxServerBlock` property to provide with an additional server block, that will instruct NGINX to use it (see `values.yaml`). Alternatively, you can specify this server block configuration using an external Secret using the property `ldapDaemon.existingNginxServerBlockSecret`.
```

Comment:
I'd would change it to a more impersonal mode. Something like:
`NGINX server need to be configured with the proxy localtion. In order to....`